nutkit/protocol/feature.py

```diff
@@ -5,4 +5,21 @@
 
 
 class Feature(Enum):
+    # === OPTIMIZATIONS ===
+    # On receiving Neo.ClientError.Security.AuthorizationExpired, the driver
+    # shouldn't reuse any open connections for anything other than finishing
+    # a started job. All other connections should be re-established before
+    # running the next job with them.
     AUTHORIZATION_EXPIRED_TREATMENT = 'AuthorizationExpiredTreatment'
```

Comment:
Since It's also an optimisation, it should have the same name scheme. We could rename the constant to `OPT_ AUTHORIZATION_EXPIRED_TREATMENT`. We could change the string itself in a next PR together with the changes on the drivers.